product_docs/docs/biganimal/release/overview/replication.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Faraway replicas"
+---
+
+You can create (faraway read) replicas of your *single node* and *high availability* clusters. In case of a failure, you can manually promote the faraway replica to a full-fledged BigAnimal cluster making it capable of accepting writes. See [Managing replicas](/biganimal/release/using_cluster/managing_replicas)
+
+## Advantages
+
+- **Disaster recovery**: You can create faraway replicas in different regions in your cloud. Deploying faraway replicas across regions can help you build a more solid disaster recovery (DR) plan.
+
+- **Improved read query performance**: If the faraway replica is set up in the same region, your application can benefit from an improved read query performance. For example, applications can read from an *us-west* replica the writes made to a cluster in the *us-east* region. 
```

Comment:
If the faraway replica is set up in the same region as your application, ....
For example, applications in us-west can read from a replica in us-west the writes made to ....

Reply:
Updated the wording